src/main/java/com/suse/salt/netapi/calls/modules/Locate.java

```diff
@@ -0,0 +1,182 @@
+package com.suse.salt.netapi.calls.modules;
+
+import com.google.gson.annotations.SerializedName;
+import com.google.gson.reflect.TypeToken;
+import com.suse.salt.netapi.calls.LocalCall;
+
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+/**
+ * salt.modules.locate
+ * Module for using the mlocate utilities.
+ */
+public class Locate {
+
+    private static final LocalCall<List<String>> VERSION = new LocalCall<>(
+            "locate.version",
+            Optional.empty(), Optional.empty(),
+            new TypeToken<List<String>>() {
+            });
+
+    private static final LocalCall<List<String>> UPDATEDB = new LocalCall<>(
+            "locate.updatedb",
+            Optional.empty(), Optional.empty(),
+            new TypeToken<List<String>>() {
+            });
+
+    private static final LocalCall<Stats> STATS = new LocalCall<>("locate.stats",
+            Optional.empty(), Optional.empty(),
+            new TypeToken<Stats>() {
+            });
+
+    /**
+     * Locate module result object
+     */
+    public static class Stats {
+        private Long files;
```

Comment:
Hello,
I usually use primitive types except in some special scenarios. With this API I just wanted to be consistent with the rest of the code. When I was developing I looked at "com.suse.salt.netapi.calls.modules.File" class and it has boxed types for the result class. But checking the rest of the code again after you comment, it is true that in other classes the primitive types are used.

So, do you want me to change it to primitive types?